src/js/helpers/cleanData.js

```diff
@@ -128,7 +128,7 @@ export const cleanTopics = allTopics => {
 
   let cleanedTopics = cleanLinks(allTopics, '/topics');
   cleanedTopics.map(topic => {
-    topic.services = cleanLinks(topic.services, '/services'); //for navigation
+    topic.services = cleanLinks(topic.servicepageSet, '/services'); //for navigation
```

Comment:
I saw this in the other PR and didn't ask but since I'm seeing it again here I will, what is a `servicepageSet`?

Reply:
This is related to Django autogenerating a look up property to other page models -- I provided a more detailed explanation in the [Joplin PR here](https://github.com/cityofaustin/joplin/pulls/166)